study-builder/studies/shared/family-history/snippets/family-history-section-instructions.conf

```diff
@@ -0,0 +1,43 @@
+{
+  "titleTemplate": null,
+  "bodyTemplate": {
+    "templateType": "HTML",
+    "templateText": """
+        <p class="ddp-section-title">$fh_instructions_section_title</p>
+        <p class="PageContent-text">$fh_instructions_p1</p>
+        <p class="PageContent-text">$fh_instructions_p2</p>
+        """,
+    "variables": [
+      {
+        "name": "fh_instructions_section_title",
+        "translations": [
+          {
+            "language": "en",
+            "text": "Instructions"
+          }
+        ]
+      },
+      {
+        "name": "fh_instructions_p1",
+        "translations": [
+          {
+            "language": "en",
+            "text": ${INSTRUCTIONS_P1_EN}
+          }
+        ]
+      },
+      {
+        "name": "fh_instructions_p2",
+        "translations": [
+          {
+            "language": "en",
+            "text": """
+              We appreciate you providing any information that you know, but it is okay if there are details you are unable to provide. If you don’t know the answers to any of the questions, <b>please select “I don’t know.”</b> """
```

Comment:
Hmmm I think different studies have different CSS when it comes to bolding text. And I vaguely recall that `<b>` is not the best practice? Might want to check CSS in various studies.

Reply:
Well,`<b>` might be an oldie, but still a very valid goodie.
But it does look like we have been using a specific css class to embolden text, so I have updated. Thanks for flagging. e3364422